src/net.c

```diff
@@ -55,17 +55,16 @@ extern unsigned long otraffic_irc_today, otraffic_bn_today, otraffic_dcc_today,
                      otraffic_unknown_today;
 
 char natip[121] = "";         /* Public IPv4 to report for systems behind NAT */
-char listen_ip[121] = "";     /* IP (or hostname) for listening sockets       */
+char listen_ip[121] = "0.0.0.0"; /* IP (or hostname) for listening sockets    */
```

Comment:
the default should be left at "", 0.0.0.0 could mean ipv4 only, depending on kernel settings